### PR TITLE
Add tests for extract_bbs_from_obj

### PR DESCRIPTION
**Pull request by boomanaiden154**
This patch adds testing infrastructure and initial tests for the extract_bbs_from_obj script to ensure that all functionality is working.

### CODE REVIEW

**Comment by boomanaiden154** [comment]
This is stacked on https://github.com/google/gematria/pull/86.

File: gematria/datasets/extract_bbs_from_obj_test.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from lit.main import main
```
Comment (ondrasej) [inline gematria/datasets/extract_bbs_from_obj_test.py:1]: Nit: The Google Python style does not allow direct imports of classes and functions.

File: gematria/datasets/extract_bbs_from_obj_test.py
Comment (ondrasej) [inline gematria/datasets/extract_bbs_from_obj_test.py:1]: Please add the Apache 2.0 license notice. Same for all the other Python files.

**Review by ondrasej — CHANGES_REQUESTED** [review]

File: gematria/datasets/extract_bbs_from_obj_test.py
Comment (boomanaiden154) [reply gematria/datasets/extract_bbs_from_obj_test.py:1]: Added.

File: gematria/datasets/extract_bbs_from_obj_test.py (hunk 1 above)
Comment (boomanaiden154) [reply gematria/datasets/extract_bbs_from_obj_test.py]: Fixed.

**Review by ondrasej — APPROVED** [review]

**Pull request merged**